web/src/beta/features/AccountAndWorkSpaceSetting/innerPages/common.tsx

```diff
@@ -0,0 +1,79 @@
+import { Collapse, Typography } from "@reearth/beta/lib/reearth-ui";
+import { useT } from "@reearth/services/i18n";
+import { styled } from "@reearth/services/theme";
+
+export const InnerPage = styled("div")<{
+  wide?: boolean;
+  transparent?: boolean;
+}>(({ wide, transparent, theme }) => ({
+  boxSizing: "border-box",
+  display: "flex",
+  width: "100%",
+  maxWidth: wide ? 950 : 750,
+  backgroundColor: transparent ? "none" : theme.bg[1],
+  borderRadius: theme.radius.normal
+}));
+
+export const InnerSidebar = styled("div")(({ theme }) => ({
+  display: "flex",
+  flexDirection: "column",
+  width: 213,
+  borderRight: `1px solid ${theme.outline.weaker}`,
+  padding: `${theme.spacing.normal}px 0`
+}));
+
+export const SettingsWrapper = styled("div")(({ theme }) => ({
+  display: "flex",
+  flexDirection: "column",
+  width: "100%",
+  flex: 1,
+  ["> div:not(:last-child)"]: {
+    borderBottom: `1px solid ${theme.outline.weaker}`
+  }
+}));
+
+export const SettingsFields = styled("div")(({ theme }) => ({
+  display: "flex",
+  flexDirection: "column",
+  gap: theme.spacing.largest
+}));
+
+export const SettingsRow = styled("div")(({ theme }) => ({
+  display: "flex",
+  justifyContent: "space-between",
+  flexDirection: "row",
+  gap: theme.spacing.largest
+}));
+
+export const SettingsRowItem = styled("div")(() => ({
+  width: "100%"
+}));
+
+export const Thumbnail = styled("div")<{ src?: string }>(({ src, theme }) => ({
+  width: "100%",
+  paddingBottom: "52.3%",
+  fontSize: 0,
+  background: src
+    ? `url(${src}) center/contain no-repeat`
+    : theme.relative.dark,
+  borderRadius: theme.radius.small
+}));
```

Comment:
```suggestion
```
Not usefull here i guess

Reply:
deleted!
https://github.com/reearth/reearth-visualizer/pull/1155/commits/bfe46f3e8a94c6c98880f376c16256a5a55c26e1